Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM node:lts-alpine
 
 RUN set -ex && mkdir /app
+RUN apk add --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN wget https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp -O /usr/local/bin/yt-dlp && chmod a+rx /usr/local/bin/yt-dlp
```

Comment:
There is a package of it for Alpine Linux, which we used it as our base image: https://pkgs.alpinelinux.org/package/edge/community/x86/youtube-dl

Reply:
This works. Thank you for the suggestion.